SCRUM-32 Add logging aspect and update logging configurations

krwro opened the pull request:
A new LoggingAspect class has been created to handle logging throughout the application. Aspect-oriented programming (AOP) dependency was added to pom.xml to support it. The logging configurations were also updated in the application properties files, differentiating log levels between the production and the development/test environments.

krwro committed:
SCRUM-32 Add logging aspect and update logging configurations

A new LoggingAspect class has been created to handle logging throughout the application. Aspect-oriented programming (AOP) dependency was added to pom.xml to support it. The logging configurations were also updated in the application properties files, differentiating log levels between the production and the development/test environments.

ovanov commented on src/main/java/ch/nexusnet/postmanager/aspect/LoggingAspect.java:
```diff
@@ -0,0 +1,40 @@
+    // Pointcut defining where the logger gets used
+    @Pointcut("within(@org.springframework.stereotype.Repository *)" +
+            " || within(@org.springframework.stereotype.Service *)" +
+            " || within(@org.springframework.web.bind.annotation.RestController *)")
```
If this Aspect is meant to be used throughout the post-manager service, you should also consider extending the point cut with an `@Component` annotation, since you are using them as well.  (Not mandatory)

krwro replied:
Good catch. I will add it. Thanks!

ovanov reviewed (approved):
Nice aspect, will apply that to the user-manager too!

krwro committed:
Extend logging aspect to Spring components

A change has been made to extend the application's logging aspect to also include Spring components.
